src/Commands/MakeModelsCommand.php

```diff
@@ -102,7 +102,10 @@ public function fire()
 
         $this->ruleProcessor = new RuleProcessor();
         $this->databaseEngine = config('database.default', 'mysql');
-        \DB::connection()->setFetchMode(\PDO::FETCH_CLASS);
+        $dispatcher = new \Illuminate\Events\Dispatcher;
```

Comment:
You're creating a new dispatcher instance so StatementPrepared event will never get fired. You need to take the dispatcher from the service container.